src/ontology/edits/oeo-social.omn

```diff
@@ -1276,6 +1279,42 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/796",
     
     SubClassOf: 
         OEO_00140040
+        
+     
+Class: OEO_00140149
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A transformative measure is a process that is an activity that has a direct impact on a specific variable in question.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/444
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/797",
+        rdfs:label "transformative measure"@en
+    
+    SubClassOf: 
+        <http://purl.obolibrary.org/obo/BFO_0000015>
+    
+    
+Class: OEO_00140150
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A policy is a specifically dependent continuant that is a deliberate system of principles, rules and guidelines, adopted by an organization to guide decision making with respect to particular situations and implemented to achieve stated goals.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/444
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/797",
+        rdfs:label "policy"@en
+    
+    SubClassOf: 
+        <http://purl.obolibrary.org/obo/BFO_0000020>
+    
+    
+Class: OEO_00140151
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A policy instrument is a specifically dependent continuant that is an action by the government that is intended to promote the adoption of a (transformative) measure?",
```

Comment:
Please delete the "?" and use a point instead.

Reply:
Thanks for spotting this.